whosehouse.html

```diff
@@ -2,6 +2,9 @@
 <html>
   <head>
     <title>Whose House?</title>
+    <script src="js/jquery.min.js"></script>
+    <script src="js/bootstrap.min.js"></script>
+    <script src="whosehouse.js"></script>
```

Comment:
The best practice is to put scripts at the end of the document rather than the beginning because it markup rendering pauses for the script execution.

Reply:
Done - I still have the explicit jquery call to $('#whose-house-button') in the HTML, not sure if that's okay.